Update move documentation (#1231)
Enalean/tuleap-documentation-en: languages/en/user-guide/trackers/usage/move-artifact.rst

   - you must be tracker administrator of both source and target trackers
   - artifact deletion must be allowed (``Site admin > Administration > Tracker > Artifacts Deletion``)

> However artifacts containing artifacts connected to FRS releases or Program cannot be moved.

Reviewer: suggestion: It sounds a bit clumsy. What do you think about the following:

However, there are some limitations:
- Artifacts linked to FRS releases cannot be moved in another tracker.
- Artifacts related to a Program project (SAFE®) cannot be moved in another tracker.
- Artifacts containing an encrypted field cannot be moved in another tracker.

I'm not sure about the third one, because it is a plugin we do not recommend. But in the other hand, people using it needs to be aware the feature will be blocked if they use these fields.

Author: I didn't add the third point, the plugin is barely usable if we see that clients complains about this limit, well document it later

.. NOTE::

>  Move is based on ducktyping, if the origin field and the destination field have the same names and types, the Artifact can be moved to the destination tracker.

Reviewer: ```suggestion
The artifact move works using duck typing. If the source field and the destination field have the same name, the same type, or are compatible (string/text, int/float), then the field value can be moved to the destination tracker.

Regarding list fields, they need to have the same bind type (static values, users, user groups). If a value does not exist in the destination list field, then it will be ignored.
```

>  If fields have different name, a modal will be displayed indicating you data you will lost

Reviewer: ```suggestion
When a field does not match these conditions, then its data will be lost.
```

>  If fields have different name, a modal will be displayed indicating you data you will lost

Reviewer: You did not explain anything about artifact links, it may be worth it to explain that the types can be cleared when the links use system natures, or a custom nature that is not used in the destination tracker + there are reverse links with the _is_child nature.

>     And follow-up comments can be moved.

Reviewer: Not sure this sentence fits well now, maybe we should move it somewhere else, or delete it.
